src/serialize.rs

```diff
@@ -0,0 +1,14 @@
+use serde_pickle::{from_reader, to_writer};
+use std::collections::BTreeMap;
+use std::fs::File;
+
+pub fn save(state_dict: BTreeMap<String, Vec<f64>>, path: &str) {
```

Comment:
@jrego43 is working on a `Model` trait and these methods might fit better there, making this comment as a future reminder